helm/templates/deployment.yaml

```diff
@@ -26,13 +26,17 @@ spec:
           secret:
             secretName: ssh-key
             defaultMode: 0400
+        - name: tmp
+          emptyDir: {}
       containers:
         - name: tunnel
           image: dajudge/tunnel-ingress:{{ .Values.tag }}
           volumeMounts:
             - name: ssh-key
               mountPath: /ssh-key/
               readOnly: true
+            - name: tmp
+              mountPath: /tmp/
```

Comment:
I am wondering if it's wise to use /tmp here. Maybe sth like /var/run/tunnel-ingress/ would be more appropriate? 🤔

I am not really certain but it might even be the case that we wouldn't even need a volume then (maybe /tmp gets special treatment?) 🤔